website/content/docs/v0.7/Learn More/upgrades.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Upgrades
```

Comment:
Sorry I missed this last time ... needs `weight`, otherwise it defaults to 0 (which I don't think we want?)